Skip symbolic links that point to a non-existent file

```
panic: open .remarkrc: no such file or directory

goroutine 1 [running]:
main.main.func1({0xc0000caa00, 0x9}, {0xb?, 0x9?}, {0xc000093d78?, 0x49f33d?})
        github.com/ianlewis/todos/internal/cmd/todos/main.go:99 +0x357
io/fs.walkDir({0x5d04e0, 0xc00021ce50}, {0xc0000caa00, 0x9}, {0x5d0948, 0xc0000b3340}, 0xc000093f50)
        io/fs/walk.go:73 +0x78
io/fs.walkDir({0x5d04e0, 0xc00021ce50}, {0x5ce500, 0x1}, {0x5d0910, 0xc00021ce60}, 0xc000093f50)
        io/fs/walk.go:95 +0x2e6
io/fs.WalkDir({0x5d04e0, 0xc00021ce50}, {0x5ce500, 0x1}, 0xc000093f50)
        io/fs/walk.go:122 +0xde
main.main()
        github.com/ianlewis/todos/internal/cmd/todos/main.go:95 +0x2c5
```

--- a/internal/cmd/todos/main.go
+++ b/internal/cmd/todos/main.go
@@ -93,7 +93,12 @@ func main() {
 
 		if fInfo.IsDir() {
 			fs.WalkDir(os.DirFS(path), ".", func(subPath string, d fs.DirEntry, err error) error {
-				fullPath := filepath.Join(path, subPath)
+				fullPath, err := filepath.EvalSymlinks(filepath.Join(path, subPath))
+				if err != nil {
+					// NOTE: If the symblink couldn't be evaluated just skip it.
+					return nil
+				}
+
 				f, err := os.Open(fullPath)
 				if err != nil {
 					panic(err)

